test/browser_tests/shared_objects/footer.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function Footer() {
+  this.get = function() {
+    browser.get( '/' );
+  };
+
+  this.footer = element( by.css( '.footer' ) );
+  this.navList = this.footer.element( by.css( '.footer_nav-list' ) );
+  this.navLinks = element.all( by.css( '.footer-middle-left a, .footer-middle-right a' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/5677a0dd00eff6540d4d132580e91eab11dbb169